packages/grafana-ui/src/components/Dropdown/ButtonSelect.tsx

```diff
@@ -56,17 +57,20 @@ const ButtonSelectComponent = <T,>(props: Props<T>) => {
       </ToolbarButton>
       {isOpen && (
         <div className={styles.menuWrapper}>
-          <ClickOutsideWrapper onClick={onCloseMenu} parent={document}>
-            <Menu>
-              {options.map((item) => (
-                <MenuItem
-                  key={`${item.value}`}
-                  label={(item.label || item.value) as string}
-                  onClick={() => onChangeInternal(item)}
-                  active={item.value === value?.value}
-                />
-              ))}
-            </Menu>
+          <ClickOutsideWrapper onClick={onCloseMenu} parent={document} includeButtonPress={false}>
+            <FocusScope contain restoreFocus autoFocus>
```

Comment:
Not sure we want `contain` here. Looking at the [WAI-ARIA authoring practices](https://www.w3.org/TR/wai-aria-practices-1.1/#menu), it says:
> Tab: Moves focus to the next element in the tab sequence, and if the item that had focus is not in a menubar, closes its menu and all open parent menu containers.
This will probably involve a change to how `Menu` handles focus/blur.

Another thing I noticed from the authoring practices guide is that the menu should open when pressing the down arrow.

Reply:
> Another thing I noticed from the authoring practices guide is that the menu should open when pressing the down arrow.

I think this is optional though.

> (Optional) Down Arrow: opens the menu and moves focus to the first menu item.

Reply:
> Not sure we want contain here

I think we do. This prevents us from moving focus to the next element in the tab sequence. So, at very point in time, refresh picker menu would always be closed when moving focus to the next tabable element.

Reply:
> I think we do. This prevents us from moving focus to the next element in the tab sequence. So, at every point in time, refresh picker menu would always be closed when moving focus to the next tabable element.

I think this is desired behaviour though?
From WAI-ARIA authoring practices:

> "Tab: Moves focus to the next element in the tab sequence, and if the item that had focus is not in a menubar, closes its menu and all open parent menu containers."

Reply:
Oh...I think I get your point now @kaydelaney 
Would look into this

Reply:
Pushed some changes already...feel free to take a look @kaydelaney